pkg/backup/backup.go

```diff
@@ -41,6 +41,9 @@ import (
 	kubeutil "github.com/heptio/ark/pkg/util/kube"
 )
 
+// BackupVersion is the current backup version for Ark.
+const BackupVersion = "1.0.0"
```

Comment:
Should probably be `1`

Reply:
https://github.com/heptio/ark/blob/c34e3b91276f7a2787264deb5f6be345990c7c8d/pkg/controller/backup_controller.go#L55